searchlib/src/vespa/searchlib/fef/indexproperties.cpp

```diff
@@ -52,8 +52,11 @@ lookupUint32(const Properties &props, const vespalib::string &name, uint32_t def
     Property p = props.lookup(name);
     uint32_t value(defaultValue);
     if (p.found()) {
-        const auto & s = p.get();
-        std::from_chars(s.c_str(), s.c_str() + s.size(), value);
+        const auto & valS = p.get();
+        const char * start = valS.c_str();
+        const char * end = start + valS.size();
+        while (isspace(start[0]) && start != end) { start++; }
```

Comment:
Consider swapping comparison order so that `start` is never dereferenced when `start == end`

Reply:
Ouch, good catch.